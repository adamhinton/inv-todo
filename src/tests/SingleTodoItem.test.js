import SingleTodoitem from "../components/SingleTodoItem";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";

const fakeTodoItemIncomplete = {
  text: "Fake Todo Name",
  id: 1,
  completed: false,
};

test("[1] SingleTodoItem incomplete renders without errors", () => {
  render(
    <SingleTodoitem completed={false} todoItem={fakeTodoItemIncomplete} />
  );
});
