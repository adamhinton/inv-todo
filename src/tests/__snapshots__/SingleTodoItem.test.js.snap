// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[4] SingleTodoItem completed = true matches snapshot from 5.29.22 1`] = `
<li
  className="sc-eCYdqJ hJzPOU"
  onClick={[Function]}
>
  <input
    data-testid="todo-checkbox"
    defaultChecked={true}
    type="checkbox"
  />
  <h4>
    Fake Todo Name Complete
  </h4>
  <button
    className="sc-hKMtZM fLGsmg"
  >
    Delete
  </button>
</li>
`;

exports[`[5] SingleTodoItem completed = false matches snapshot from 5.29.22 1`] = `
<li
  className="sc-eCYdqJ hJzPOU"
  onClick={[Function]}
>
  <input
    data-testid="todo-checkbox"
    defaultChecked={true}
    type="checkbox"
  />
  <h4>
    Fake Todo Name Incomplete
  </h4>
  <button
    className="sc-hKMtZM fLGsmg"
  >
    Delete
  </button>
</li>
`;
